daisy_workflows/build-publish/rhui/health_check.py

```diff
@@ -101,37 +140,21 @@ def main(node_type: str, result_file: typing.TextIO, nfs_mount: str):
     type=argparse.FileType("w"),
     help="file to write result",
   )
-  parser.add_argument(
-    "--nfs_mount",
-    dest="nfs_mount",
-    type=str,
-    nargs=1,
-    required=True,
-    help="directory of NFS mount",
-  )
-  parser.add_argument(
-    "--log",
-    dest="log",
-    type=str,
-    nargs=1,
-    choices=["stdout", "syslog"],
-    default="syslog",
-    help="where to write logs",
-  )
   args = parser.parse_args()
-  if args.log[0] == "stdout":
-    handler = logging.StreamHandler(sys.stdout)
-  else:
-    handler = logging.handlers.SysLogHandler(address="/dev/log")
-    # syslog parses on the colon to determine the tag for the message.
-    handler.ident = "rhui-health-check: "
+
+  handler = logging.StreamHandler(sys.stdout)
   logging.basicConfig(
     level=logging.DEBUG,
     format="%(name)s - %(levelname)s - %(message)s",
     handlers=[handler],
   )
-  main(
-    node_type=args.node[0],
-    result_file=args.result_file,
-    nfs_mount=args.nfs_mount[0],
-  )
+
+  args.result_file.truncate()
+  if HealthCheckSuite(args.node):
+    args.result_file.write("HEALTHY\n")
+  else:
+    args.result_file.write("ERROR\n")
```

Comment:
While you're here, I was thinking it'd be helpful to use `exit(1)` when there's a failure so that systemd's logs show there's a failure for the overall unit. If I remember correctly, in the serial output logs for the instance, when the health check fails, it looks like it's passing since systemd reports that the unit was successful.

Reply:
the health check *is* passing, though. it is correctly running and determining the health status of the node, which is its job. what are the scenarios where you'd want systemd to see the health check itself as failed? does it have some impact on the timer?